test/monitor/monitor.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
```

Comment:
Can we turn this into a make target and call it in the 'Monitor Deployment' stage of the pipeline please?